$OpenBSD$
unbreak clang build
error: '__sF' declared as array of references of type '::FILE *(&)' (aka '__sFILE *&')

Index: src/test/test_stubgenerator.cpp
--- src/test/test_stubgenerator.cpp.orig
+++ src/test/test_stubgenerator.cpp
@@ -20,23 +20,24 @@
 
 #include <sstream>
 
+
 using namespace jsonrpc;
 using namespace std;
 
 namespace teststubgen {
 struct F {
-  FILE *stdout;
-  FILE *stderr;
+  FILE *pStdout;
+  FILE *pStderr;
   vector<StubGenerator *> stubgens;
   vector<Procedure> procedures;
   F() {
-    stdout = fopen("stdout.txt", "w");
-    stderr = fopen("stderr.txt", "w");
+    pStdout = fopen("stdout.txt", "w");
+    pStderr = fopen("stderr.txt", "w");
   }
 
   ~F() {
-    fclose(stdout);
-    fclose(stderr);
+    fclose(pStdout);
+    fclose(pStderr);
   }
 };
 } // namespace teststubgen
